docs/app/Examples/elements/Input/Variations/Fluid.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import {Input} from 'stardust';
+
+export default class Fluid extends Component {
+  render() {
+    return (
+      <Input className='fluid icon' placeholder='Search...'>
+        <i className='search icon' />
+      </Input>
```

Comment:
Same issue as the "Labeled.js". Button is only displaying to the right of the input field